packages/cozy-stack-client/src/NotesCollection.js

```diff
@@ -47,6 +53,26 @@ class NotesCollection extends DocumentCollection {
       data: { ...normalizeNote(resp.data), _deleted: true }
     }
   }
+
+  /**
+   * open - Returns the details to build the note's url
```

Comment:
```suggestion
   * Returns the details to build the note's url
```

Reply:
I'm just following the naming scheme of the comments in this file. Should I remove the name of the methods in all of them?